ch3/Makefile

```
@@ -1,13 +1,15 @@
 init:
 	pip install -U pip
-	pip install boto3==1.26.8 mlflow==1.30.0 scikit-learn
+	pip install boto3==1.26.8 mlflow==1.30.0 scikit-learn pandas
 
 server:
 	docker-compose up -d
 
 server-clean:
 	docker-compose down -v
 	docker rmi ch3-mlflow-server
+	docker rmi minio/minio
```

```suggestion
	docker rmi ch3-mlflow-server minio/minio
```
이렇게 해본지가 오래되서 그런데 한줄로 가능하던가요? 된다면 한줄로 바꾸면 좋을 것 같아요

---

네 동민님 시도해보니 한 줄로도 가능합니다! 한 줄로 수정하겠습니다 :) 좋은 정보 감사해요 😄